src/schema.rs

```diff
@@ -0,0 +1,69 @@
+//! Schema
+
+#![allow(missing_docs)]
+use template;
+use std::collections::BTreeMap;
+use serde_json::Value;
+
+/// Schema
+pub struct Schema {
+    files: Vec<File>,
```

Comment:
If the schema is a vector of files, does it mean that the name of the file (which acts as a key in the JSON representation) gets lost when converting from JSON to `Schema`? If so, how would we resolve references between files?